spec/factories/users.rb

```diff
@@ -1,6 +1,7 @@
 FactoryBot.define do
-  factory :user do
-    email "test@test.com"
-    password "1234567"
+  factory :user do |u|
+    u.sequence(:email) { |n| "journalist#{n}@test.com"}
+    u.password "password123"
+    u.role 'journalist'
```

Comment:
Is the journalist the most common role. Also, why the change in notation?

Reply:
changed to `subscriber`